spec/features/inputs/tags_input_spec.rb

```diff
@@ -43,25 +45,26 @@
     end
 
     it "shows preloaded items", js: true do
-      expect_select2_options_count_to_eq(3)
+      expect_slimselect_options_count_to_eq(3)
     end
 
     context "with added item" do
-      before { pick_select2_entered_option(@item1.name) }
+      before { pick_slimselect_entered_option(@item1.name) }
 
       it "adds/removes hidden item", js: true do
         item_id = "#invoice_item_ids_#{@item1.id}"
         input = find(item_id, visible: false)
         expect(input.value).to eq(@item1.id.to_s)
         expect(input[:name]).to eq("invoice[item_ids][]")
-        find(".select2-selection__choice__remove").click
+        find(".ss-value-delete").click
+        sleep 0.5
         expect { find(item_id, visible: false) }.to raise_error(Capybara::ElementNotFound)
       end
 
       it "does not allow new items", js: true do
-        expect_select2_choices_count_to_eq(1)
-        fill_select2_input_and_press_return("Not preloaded item")
-        expect_select2_choices_count_to_eq(1)
+        expect_slimselect_options_count_to_eq(3)
+        expect { add_slimselect_option("Not preloaded item") }.to raise_error(Capybara::ElementNotFound)
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is too long. [104/100]